onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -63,11 +63,22 @@ def check_and_normalize_provider_args(
     provider_name_to_options = collections.OrderedDict()
 
     def set_provider_options(name, options):
+        if (
+            os.environ.get("ORT_FALLBACK_CUDA_EP_TO_ROCM_EP", "0") == "1"
+            and "CUDAExecutionProvider" not in available_provider_names
```

Comment:
```suggestion
            and "CUDAExecutionProvider" not in available_provider_names
            and "ROCMExecutionProvider" in available_provider_names
```
should you check whether rocm is an option before using it?